src/schema/axe.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
The schema files in this directory are meant to be used to define the structure of the JCAD file. Meanwhile `grid` and `axes` are GUI-related things,  I think we might want to separate these two and put the GUI schemas elsewhere.